File: 01-Assignment-02-02-2026/Using_C/program_12.c
/* Program 12: Find the sum of digits of an integer using while loop */

#include <stdio.h>

int main() {
    int num,sum = 0;
    printf("Enter your Number : ");
    scanf("%d",&num);
    if (num < 0) 
        num = -num;
    while (num > 0)
    {
        sum+=(num%10);
        num/=10;
    }
    printf("Sum of Digits of Given number is : %d",sum);
    


    return 0;
}

File: 01-Assignment-02-02-2026/Using_Java/Program12.java
/*
Program 12
Java program to find the sum of the digits of an integer using while loop
*/

import java.util.Scanner;

public class Program12 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter an Integer : ");
        int number = scanner.nextInt();

        // System.out.println("Original Number : " + number);
        int sum = 0;
        while (number > 0) {
            sum += number % 10;
            number /= 10;
        }
        System.out.println("Sum of digits : " + sum);
        scanner.close();
    }

}
